### ISSUE TITLE
Installation via Luarocks fails with " Error: Couldn't extract archive: unrecognized filename extension"

### ISSUE THREAD

**Issue by crater2150** [stefano-m/lua-dbus_proxy#11]
Installing lua-dbus_proxy via the luarocks cli results in an error:
```
 % luarocks install --local dbus_proxy 
Installing https://luarocks.org/dbus_proxy-0.10.3-2.src.rock

Error: Couldn't extract archive /tmp/luarocks_dbus_proxy-0.10.3-2-6sTEvt/lua-dbus_proxy: unrecognized filename extension
```
It only happens for the two 0.10.3 versions, and only when installing from luarocks.org with `luarocks install`. Manually installing with `luarocks make rockspec/dbus_proxy-0.10.3-2.rockspec` works fine, so it seems the packages on luarocks.org are broken.

**Comment by stefano-m** [comment]
Hi there, thanks for reporting the issue. I am using the package outside luarocks and had not noticed.

This appears to be a packaging error when I switched the rocspeck URL from SSH to HTTP. I am suprised that luarocks created a "valid" package though.

For the time being, I have removed the badly packaged versions from luarocks.

When I have time, I will try and figure out what's going on and repackage the latest version.

I will keep this issue open as a placeholder.

**Comment by stefano-m** [comment]
Hi @crater2150 

I think I fixed this issue in 0f99607e22df3880968a88e58b380b879d36694d

I have rebuilt the rock and inspected it and it seems to have the correct contents.

Thank you again for raising the problem

I am closing this issue, but if this is not solved on your side, feel free to add a comment here and we'll reopen it.
